### ISSUE TITLE
Wired things, module must have its parameters and buffers on device cuda:0 (device_ids[0]) but found one of them on device: cpu

### ISSUE THREAD

**Issue by kehuantiantang** [vacancy/Synchronized-BatchNorm-PyTorch#37]
I use pytorch 1.2.0, I first init the model, then use `model.cuda()` to put the model into gpu, then, I call 
`model = nn.DataParallel(model)
model = convert_model(model)
`
train the model, give me such error, could you give me some information to avoid this ? thank you

**Comment by vacancy** [comment]
I think you should call model.cuda() after your `model = convert_model(model)`
